lib/active_flag.rb

```diff
@@ -34,14 +34,18 @@ def flag(column, keys)
       end
 
       # Scopes
-      define_singleton_method "where_#{column}" do |*args|
-        options = args.extract_options!
-        integer = active_flags[column].to_i(args)
-        column_name = connection.quote_table_name_for_assignment(table_name, column)
-        if options[:op] == :and
-          where("#{column_name} & #{integer} = #{integer}")
-        else
-          where("#{column_name} & #{integer} > 0")
+      [nil, :not_].each do |negation|
```

Comment:
This approach actually increases McCabe complexity — I prefer to have a separate `define_singleton_method` for `where_not` for clarity.

Reply:
@kenn Done